content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst

```diff
@@ -0,0 +1,233 @@
+Products`.
+
+.. figure:: type/on-hand.png
+   :align: center
+   :alt: Show "On Hand" and "Forecast" smart buttons.
+
+   Current and forecasted quantities displayed in the **On Hand** and **Forecasted** smart buttons
+   on the product form.
+
+.. _inventory/product_management/on-hand-con:
+
+On the other hand, consumable products are regarded as always available, and they **cannot** be
+managed using reordering rules or lot/serial numbers.
+
+.. _inventory/product_management/transfer-store:
+
+Create transfer
+---------------
+
+*Transfers* are any warehouse operation, such as receipts, internal or batch transfers, or
+deliveries.
+
+When creating a transfer for storable products in the *Inventory* app, transfers modify the on-hand
+quantity at each location.
+
+For example, transferring five units from the internal location `WH/Stock` to `WH/Packing Zone`
+decreases the recorded quantity at `WH/Stock` and increases it at `WH/Packing Zone`.
+
+.. _inventory/product_management/transfer-con:
+
+For consumable products, transfers can be created, but exact quantities at each storage location are
+not tracked.
+
+.. _inventory/product_management/transfer-serv:
+
+Service products cannot be included in transfers, but these products can be `linked to projects and
+tasks for deadline tracking <https://www.youtube.com/watch?v=fix2LGkv13c>`_.
+
+.. _inventory/product_management/po:
+
+Create purchase order
+---------------------
+
+Both storable and consumable products can be included in a request for quotation in the *Purchase*
+app.
+
+However, when receiving consumable products, their on-hand quantity will not change upon validating
+the receipt (e.g. `WH/IN`).
+
+.. _inventory/product_management/manufacture:
+
+Manufacture or subcontract
+--------------------------
+
+Storable and consumable products can both be manufactured, subcontracted, or included in a bill of
+materials (BoM).
+
+.. figure:: type/manufacture.png
+   :align: center
+   :alt: Show "Bill of Materials" and "Used In" smart buttons.
+
+   When the **Bill of Materials** and **Used In** smart buttons are visible on the product form,
+   this indicates the product can be manufactured or used as a component of a |BOM|.
+
+.. _inventory/product_management/package:
+
+Packages
+--------
+
+Both storable and consumable products can be placed in packages.
+
+.. seealso::
+   :doc:`package`
+
+However, for consumable products, the quantity is not tracked, and the product is not listed in the
+package's :guilabel:`Contents`, accessed by going to :menuselection:`Inventory app --> Products -->
+Packages` and selecting the desired package.
+
+.. figure:: type/package-content.png
+   :align: center
+   :alt: Show Packages page, containing the package contents list.
+
+   A consumable product was placed in the package, but the **Content** list does not list it.
+
+If the *Move Entire Package* feature is enabled, moving a package updates the location of the
```

Comment:
If I'm understanding correctly this feature is italicized because it's not immediately on the UI and is simply being used in a definition-like way. Is that right? 😄

Reply:
Yes! The term is italicized because the feature is being referred to, but it's currently not displayed on the UI